doc/config_rsmtool.rst

```diff
@@ -185,6 +185,17 @@ standardize_features *(Optional)*
 If this option is set to ``false`` features will not be standardized by dividing by the mean and multiplying by the standard deviation. Defaults to ``true``.
 
 
+.. _use_thumbnails_rsmtool:
```

Comment:
Perhaps move this next to other report-related fields (section lists)?

Reply:
Sure, I can rearrange these in my `specify-features-as-list` branch, since I still need to update the documentation there.